src/OpenIddict.Server/OpenIddictServerBuilder.cs

```diff
@@ -157,7 +157,9 @@ public OpenIddictServerBuilder AddEncryptionCredentials(EncryptingCredentials cr
         /// <summary>
         /// Registers an encryption key.
         /// </summary>
-        /// <param name="key">The security key.</param>
+        /// <param name="key">The security key. If a SymmetricSecurityKey, create using a 32-character
```

Comment:
Technically, this comment is not 100% exact, as the check doesn't only applies to `SymmetricSecurityKey` instances (e.g you could have a `JsonWebKey` instance representing a symmetric key). Let's revert this change, please.